superset/templates/appbuilder/menu.html

```diff
@@ -29,7 +29,7 @@
     {% endif %}
 {% endfor %}
 
-<script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```

Comment:
hmnn why the original superset works with this jquery?

Reply:
original superset works with jquery 1.10.2, it uses function .size() that have been removed in latest versions, so i have replace it with the version available in the appbuilder.

Reply:
This is version I added in the development process